<ui version="4.0">
<class>FinalWindow</class>
<widget class="QWidget" name="FinalWindow">
<property name="geometry">
<rect>
<x>0</x>
<y>0</y>
<width>1280</width>
<height>720</height>
</rect>
</property>
<property name="windowTitle">
<string>Space Invaders - Fim de Jogo</string>
</property>
<property name="styleSheet">
<string notr="true"/>
</property>
<widget class="QWidget" name="finalPane" native="true">
<property name="geometry">
<rect>
<x>0</x>
<y>0</y>
<width>1280</width>
<height>720</height>
</rect>
</property>
<property name="styleSheet">
<string notr="true">#finalPane {
background-image: url("sprites/background.png");
background-position: center;
}</string>
</property>
<widget class="QLabel" name="labelVitoria">
<property name="geometry">
<rect>
<x>350</x>
<y>200</y>
<width>600</width>
<height>60</height>
</rect>
</property>
<property name="font">
<font>
<pointsize>48</pointsize>
</font>
</property>
<property name="styleSheet">
<string notr="true">color: #00FF00;</string>
</property>
<property name="text">
<string>VITÃ“RIA!</string>
</property>
<property name="alignment">
<set>Qt::AlignCenter</set>
</property>
</widget>
<widget class="QLabel" name="labelDerrota">
<property name="geometry">
<rect>
<x>350</x>
<y>200</y>
<width>600</width>
<height>60</height>
</rect>
</property>
<property name="font">
<font>
<pointsize>48</pointsize>
</font>
</property>
<property name="styleSheet">
<string notr="true">color: #FF0000;</string>
</property>
<property name="text">
<string>DERROTA!</string>
</property>
<property name="alignment">
<set>Qt::AlignCenter</set>
</property>
</widget>
<widget class="QPushButton" name="playAgainButton">
<property name="geometry">
<rect>
<x>470</x>
<y>479</y>
<width>340</width>
<height>39</height>
</rect>
</property>
<property name="font">
<font>
<pointsize>24</pointsize>
</font>
</property>
<property name="text">
<string>Jogar Novamente</string>
</property>
</widget>
<widget class="QPushButton" name="exitButton">
<property name="geometry">
<rect>
<x>470</x>
<y>569</y>
<width>340</width>
<height>39</height>
</rect>
</property>
<property name="font">
<font>
<pointsize>24</pointsize>
</font>
</property>
<property name="text">
<string>Sair</string>
</property>
</widget>
</widget>
</widget>
<resources/>
<connections/>
</ui>